### PR TITLE
Bump Dashboard to Dapr 1.13 and recent client-go

### PR DESCRIPTION
**Pull request by SantoDE**
With the current Dashboard version, I came across an issue. Once a sidecar had the `--enable-api-logging` flag set, It wouldn't show up in the dashboard anymore because of an issue in the dependency code of Dapr CLI with version 1.10. To fix that, I bumped the dependencies and the related dependencies.

Since this is my first PR, happy to walk it through with the maintainers team to learn.

### CODE REVIEW

File: pkg/configurations/configurations.go [hunk 1]
```diff
@@ -201,11 +202,11 @@ func (c *configurations) getDockerComposeConfigurations(scope string) []Configur
 				Age:             age.GetAge(info.ModTime()),
 				TracingEnabled:  tracingEnabled(comp.Spec.TracingSpec.SamplingRate),
 				SamplingRate:    comp.Spec.TracingSpec.SamplingRate,
-				MetricsEnabled:  comp.Spec.MetricSpec.Enabled,
+				MetricsEnabled:  *comp.Spec.MetricSpec.Enabled,
 				MTLSEnabled:     comp.Spec.MTLSSpec.Enabled,
 				WorkloadCertTTL: comp.Spec.MTLSSpec.WorkloadCertTTL,
 				ClockSkew:       comp.Spec.MTLSSpec.AllowedClockSkew,
-				Manifest:        content,
+				Manifest:        comp.String(),
```
Comment (yaron2) [inline pkg/configurations/configurations.go:209]: Why this change here?

Comment (SantoDE) [reply pkg/configurations/configurations.go:209]: Maybe Im misinterpreting things, but pre bump `config.LoadStandaloneConfiguration(path)` returned the content as string together with the actual component. Now, after bump, this return param has gone so I needed to use the string method. Am I overlooking something?

Comment (yaron2) [reply pkg/configurations/configurations.go:209]: Ok thanks for pointing that out!

**Review by yaron2 — APPROVED** [review]
